dragonfly/block/ice.go

```diff
@@ -0,0 +1,44 @@
+package block
+
+import "github.com/df-mc/dragonfly/dragonfly/item/tool"
+
+// TODO: Slipperiness, melting and SilkTouch
+
+// Ice is a solid block similar to packed ice.
+type Ice struct {
+	noNBT
+	transparent
```

Comment:
`transparent` returns a LightDiffusionLevel of 0, but the method below returns 2 (and correctly so). I don't think you'd want to embed transparent.